### PR TITLE
Adding counter id attribute for mpls insegment

### PR DESCRIPTION
**Pull request by msomasundaran**
Signed-off-by: Midhun Somasundaran <msomasundaran@fb.com>

Adding a counter id attribute for mpls inseg similar to existing counter id for ip routes

### CODE REVIEW

**Comment by msomasundaran** [comment]
resolves #1230

**Comment by rlhui** [comment]
@qbdwlr , @JaiOCP , @marian-pritsak , @ashutosh-agrawal - would you please help review/sign-off?

File: inc/saimpls.h [hunk 1]
```diff
@@ -127,6 +127,19 @@ typedef enum _sai_inseg_entry_attr_t
      */
     SAI_INSEG_ENTRY_ATTR_NEXT_HOP_ID,
 
+    /**
```
Comment (ashutosh-agrawal) [inline inc/saimpls.h:130]: Can we add this as the last attribute in the list (between pop_qos_mode and end).

Comment (msomasundaran) [reply inc/saimpls.h]: Moved to end of list. Thanks for the comment

**Review by ashutosh-agrawal — APPROVED** [review]

**Review by kcudnik — APPROVED** [review]

**Pull request merged**